docs/greek.rst

```diff
@@ -357,7 +357,11 @@ See also `Text Cleanup <http://docs.cltk.org/en/latest/greek.html#text-cleanup>`
 Corpus Readers
 ==============
 
-Most users will want to access words, sentences, paragraphs and even whole documents via a CorpusReader object. All Corpus contributors should provide a suitable reader. There is one for Perseus Greek, and others will be made available. The CorpusReader methods: ``paras()`` returns paragraphs, if possible; ``words()`` returns a generator of words; ``sentences`` returns a generator of sentences; ``docs`` returns a generator of Python dictionary objects representing each document.
+When using CLTK to work with text from a corpus, it is advantageous to have one's code read the text through a standardized interface, rather than cutting and pasting by hand or reading it by directly opening a file in python. Using the CorpusReader class for this purpose avoids having to hardcode snippets of text or path names of files in your code. A CorpusReader object also provides facilities for breaking the text up into chunks of various sizes: documents, paragraphs, sentences, and words. These chunks are read from the corpus and returned as arrays. Specifying indices in these arrays is somewhat similar to the traditional practice of referring to parts of a text by citation styles such as Iliad Δ.124, but avoids the nuances and inconsistencies that come up because of issues such as differences between editions.
+
+All corpus contributors should provide a suitable reader. Currently there is CorpusReader in the Latin and Greek Tesserae and Perseus corpora, and the Latin Library.
```

Comment:
In principle you're right, however this update would effectively promote the corpus readers to "production", something that our text databases, and maybe the readers themselves, may not be ready for.

Let me think about this a few days and meanwhile others should chime in.

Reply:
@kylepjohnson : 'this update would effectively promote the corpus readers to "production", something that our text databases, and maybe the readers themselves, may not be ready for.'

Sorry, I don't understand what you mean by this. The change just consists of rewriting a couple of sentences of documentation in an effort to make them clearer.

Reply:
> All corpus contributors should provide a suitable reader

It's this point … which upon rereading I see was already in the docs. Let's hear from @diyclassics and @todd-cook 's take on this, in general. For context @bcrowell there has been some talk in the past about how many corpora we should be hosting ourselves and, either way, how closely we should couple our code to them. Todd's perspective, I think, has been that we should have at least one well-supported for each language. This is a fine idea, however I am leery of the project over-committing in this area.

They haven't chimed in on the past issue so let's just wait a couple days.